### PR TITLE
Makes API Tokens configurable in duration with default 365 days.

### PR DESCRIPTION
**Pull request by andricDu**

### CODE REVIEW

**Comment by andricDu** [comment]
https://github.com/overture-stack/ego/issues/312

File: src/test/java/bio/overture/ego/controller/TokenControllerTest.java [hunk 1]
```diff
@@ -84,6 +84,17 @@ public void issueTokenShouldRevokeRedundantTokens() {
...
+    log.info(response.getBody());
+
+    //    val response2 =
+    //      restTemplate.exchange(
+    //        createURLWithPort("/o/check_token?token=" + tokenRevoke.getName()),
+    //        HttpMethod.POST,
+    //        entity,
+    //        String.class,
+    //        user.getId().toString());
+    //    log.info(response2.getBody());
```
Comment (rtisma) [inline src/test/java/bio/overture/ego/controller/TokenControllerTest.java]: delete this comment. If its needed leave a todo

**Comment by rtisma** [comment]
test me please

**Comment by rtisma** [comment]
ok to test

**Comment by rtisma** [comment]
test this please

**Comment by rtisma** [comment]
@andricDu increase the timeout for the UI tests. If it times out again after this increase, then we can investigate the root cause

**Pull request merged**